src/main/java/frc/robot/sensors/imu/Lsm9ds1Imu.java

```diff
@@ -27,10 +33,21 @@ public Lsm9ds1Imu(Axis yawAxis) {
     this(yawAxis, AhrsAlgorithm.Complementary);
   }
 
+  public void recalibrate() {
```

Comment:
Just for continuity's sake I would probably put this after the table declaration.